microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -147,11 +148,11 @@ If you have an online-only deployment scenario where users are assumed to have i
         HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\EdgeUpdate\Clients\{F3017226-FE2A-4295-8BDF-00C3A9A7E4C5}
         ```
 
-    *   Alternatively, run [GetAvailableCoreWebView2BrowserVersionString](/microsoft-edge/webview2/reference/win32/webview2-idl#getavailablecorewebview2browserversionstring) and evaluate whether the `versionInfo` is `NULL`.  `NULL` indicates that the Runtime isn't installed.
+    *   Alternatively, run [GetAvailableCoreWebView2BrowserVersionString](/microsoft-edge/webview2/reference/win32/webview2-idl#getavailablecorewebview2browserversionstring) and evaluate whether the `versionInfo` is `nullptr`.  `nullptr` indicates that the Runtime isn't installed. Note that the API will return version info for non-Stable Edge channels as well if installed, and non-Stable Edge channels can be used for WebView2.
 
 1.  If the Runtime is not installed, in your app setup process, use the link (from the **Get the Link** button on the download page) to programmatically download the WebView2 Runtime Bootstrapper.
 
-1.  Invoke the WebView2 Runtime Bootstrapper from an elevated process or command prompt by issuing the command `MicrosoftEdgeWebview2Setup.exe /silent /install`.
+1.  Invoke the WebView2 Runtime Bootstrapper by issuing the command `MicrosoftEdgeWebview2Setup.exe /silent /install`. Note that running the commmand from an elevated process or command prompt will trigger a per-system install, otherwise a per-machine install will take place and may be automatically replaced by a per-system install if a per-system Edge Updater (which should come with a non-Canary Edge browser) is in place.
```

Comment:
```suggestion
1.  Invoke the WebView2 Runtime Bootstrapper by issuing the command `MicrosoftEdgeWebview2Setup.exe /silent /install`. Running that command from an elevated process or command prompt will trigger a "per-system" install, otherwise a "per-machine" install will take place.  A "per-machine" install will be automatically replaced by a "per-system" install if a per-system Edge Updater is in place.  A per-system Edge Updater is provided as part of Microsoft Edge, except for Canary.
```
"per-user" install, or "per-machine" install?